### ISSUE TITLE
Windows Store/Phone & Mac

### ISSUE THREAD

**Issue by Zambiorix** [perpetual-mobile/SimpleStorage#3]
Great component! Love it and use it a lot!
It would be truly great if there eventually was a Windows Store/Phone and a Mac version too...
I would use it even more!
Thanks and keep up the good work!

Gerd

**Comment by ronpf** [comment]
Agreed. I'm using the iOS and Android version but cannot add Simple Storage to my new WinPhone and UWP projects of the same solution.
